dace/codegen/sdfgcc.py

```diff
@@ -4,24 +4,39 @@
 import dace
 import os
 import sys
+import argparse
+import shutil
 
 
 def main():
-    if len(sys.argv) != 2:
-        print('USAGE: sdfgcc <PATH TO SDFG FILE>')
-        exit(1)
+    # Command line options parser
+    parser = argparse.ArgumentParser(description='Simple SDFG command-line compiler.')
+
+    # Required argument for SDGF file path
+    parser.add_argument('SDFGfilepath', help='<PATH TO SDFG FILE>', type=str)
```

Comment:
Please change to something more in line with the python coding guidelines. `filepath` works well.